data/sql/updates/pending_db_world/rev_1668938810265176700.sql

```diff
@@ -0,0 +1,237 @@
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Mensaje de Loken recibido', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12203;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Hugh Copa interrogado', `ObjectiveText2` = 'Gavrock interrogado', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12204;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Orbe utilizado bajo el Corazón de Vordrassil', `ObjectiveText2` = 'Orbe utilizado bajo la Extremidad de Vordrassil', `ObjectiveText3` = 'Orbe utilizado bajo las Lágrimas de Vordrassil', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12213;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Historia de Orsonn', `ObjectiveText2` = 'Historia de Kodian', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12231;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Ursoc purificado', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12236;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Trituradora entregada', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12270;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Unidades de la Alianza eliminadas', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12284;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Hostigadores heridos sanados', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12288;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Alianza eliminada en Bahía Ventura', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12317;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Rezagados de Ventura y Cía. ahuyentados', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12324;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Caballos mesteños de las Tierras Altas espantados', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12415;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Vista de águila imperial unida', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12453;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Huye de Arroyoplata', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 13524;
```

Comment:
Missing new line at the end of the file.

Reply:
> Missing new line at the end of the file.

Sry, i hope all is ok now